TIPS_and_FAQ.md

```diff
@@ -2,6 +2,47 @@
 
 ### As a user, how can I...
 
+#### Embedded NodeJS
```

Comment:
It's not really "as a user", but more "as an integrator".
I believe it would be more interesting to put it in a README.md directly inside org.eclipse.wildwebdeveloper.embedder.node

Reply:
+1

Reply:
I've added README.md to `o.e.w.embedder.node`.